lua/packer/plugin_utils.lua

```diff
@@ -163,7 +163,7 @@ plugin_utils.load_plugin = function(plugin)
           error(files)
         end
       elseif #files > 0 then
-        for _, file in ipairs(files) do vim.cmd('silent exe "source ' .. file .. '"') end
+        for _, file in ipairs(files) do file = file:gsub("\\", "/") vim.cmd('silent exe "source ' .. file .. '"') end
```

Comment:
Nitpick: if this `for` loop is no longer a single statement, then it should be split to multiple lines.

Also: I'm surprised that replacing with `/` works on Windows - is there somewhere in the Vim/Neovim help that indicates why this is OK?

Reply:
Applied the reformat myself in 53b7e1d.